platform/security/idp/security-idp-client/src/test/java/org/codice/ddf/security/idp/client/LogoutRequestServiceTest.java

```diff
@@ -113,6 +115,11 @@ public void setup() {
         encryptionService = mock(EncryptionService.class);
         session = mock(HttpSession.class);
         securityTokenHolder = mock(SecurityTokenHolder.class);
+        Subject subject = mock(Subject.class);
+        when(subject.hasRole(anyString())).thenReturn(false);
```

Comment:
This test is not intended to test security auditing, this is only added so it does not produce a null exception when pulling the subject out of the ThreadContext.